cmd/deploy/command_configuration.go

```diff
@@ -178,9 +180,8 @@ func switchRepoSchemaToHTTPS(repo string) (*url.URL, error) {
 		return repoURL, nil
 	case httpsScheme:
 		return repoURL, nil
-	default:
```

Comment:
I would keep the default and log it instead of returning the error in case we need it to debug upcoming bugs